lib/src/ascii_tree.dart

```diff
@@ -165,7 +166,7 @@ void _draw(
   if (name != null) _drawLine(buffer, prefix, isLast, name, depth <= 1);
 
   // Recurse to the children.
-  final childNames = ordered(children.keys);
+  final childNames = children.keys.sorted();
```

Comment:
Or just use `.toList()..sort();`.

It's the same thing, and you don't need an extra import.

_(Or you could have used a sorted map to begin with. If only we had one that was more efficent than `SplayTreeMap`.)_